net_drivers/udp.h

```diff
@@ -684,10 +684,7 @@ static int NBN_UDP_CliSendPacket(NBN_Packet *packet)
 
 void NBN_UDP_Register(void)
 {
-    NBN_Driver_Register(
-        NBN_UDP_DRIVER_ID,
-        NBN_UDP_DRIVER_NAME,
-        (NBN_DriverImplementation){
+    NBN_DriverImplementation driver_impl = {
```

Comment:
Could you do the same thing for the other drivers as well?

Reply:
I've implemented the same fix to the other drivers. These changes are in commit [4e807aa](https://github.com/nathhB/nbnet/pull/36/commits/4e807aa0005e55428410291653a693efb7dee414).